packages/camera/camera/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.8.2
```

Comment:
Shouldn't this be a patch update (as mentioned in the [Package versioning](https://dart.dev/tools/pub/versioning#semantic-versions) docs)?

```suggestion
## 0.8.1+1
```